source/connection/specify-connection-options/security-connection-options.txt

```diff
@@ -0,0 +1,20 @@
+.. _security-connection-settings:
+
+============================
+Connection Security Settings
+============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+MongoDB supports many options for securing your data before and during
+transportation. For information about security options, see our :ref:`Security
+section <java-security>`, which includes :ref:`TLS/SSL <tls-ssl>` and :ref:`SOCKS5 Proxy
```

Comment:
S: the section also includes authentication (SCRAM, X.509, etc) options; you can also list those here, or just remove the list and keep it as:
```suggestion
section <java-security>`.
```